pallets/handles/src/handles-utils/src/tests/constants_tests.rs

```diff
@@ -0,0 +1,17 @@
+#[path = "../../constants.rs"]
+mod constants;
+use constants::*;
+
+// Use this to regenerate the compacted ALLOWED_UNICODE_CHARACTER_RANGES.
```

Comment:
This is a tedious, semi-manual way of doing it, but I decided it was the best balance between 100% manual which was resulting in lots of mistakes, and spending a bunch of time writing a fully automated file generation script for something we will change only rarely.